### PR TITLE
added promo link

### PR DESCRIPTION
**Pull request by alexisBimatovs**
https://www.pivotaltracker.com/story/show/132235123

### CODE REVIEW

File: app/views/forever_style_guide/sections/components/navigation/_nav_products_dropdown.erb [hunk 1]
```diff
@@ -125,6 +125,12 @@
 
         <ul class="list-unstyled">
           <li>
+            <%= link_to promotions_url, class: 'dropdown-menu-nav-link', title: 'Photo Printing, Digital Art, and Storage Promotions' do %>
```
Comment (schweitzdesign) [inline app/views/forever_style_guide/sections/components/navigation/_nav_products_dropdown.erb]: promotions_url is still pointed at '/holidays' in the application helper

**Comment by schweitzdesign** [comment]
This also requires a version bump.

**Comment by schweitzdesign** [comment]
Once the above changes are made, can you close this PR and re-open against master?  And then we'll be in the habit of PRing against master v. master-3.0

File: app/views/forever_style_guide/sections/components/navigation/_nav_products_dropdown.erb [hunk 2]
```diff
@@ -125,6 +125,12 @@
...
+              <i class='fa fa-star'></i>Current Promotions
+            <% end %>
+          </li>
+
```
Comment (katywatkins) [inline app/views/forever_style_guide/sections/components/navigation/_nav_products_dropdown.erb]: extra spaces!

**Comment by katywatkins** [comment]
Everything looks good to me, just some extra spaces on one line, if you have time to fix that real quick.

**Pull request merged**